sources/engine/Stride.DotRecast/Stride.DotRecast/Stride.DotRecast.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
When importing stride props, the SDK template shouldn't be used. Otherwise, some properties might not be set properly.